carbonmark/src/Projects.ts

```diff
@@ -11552,4 +11552,16 @@ export const PROJECT_INFO: ProjectInfo[] = [
     '0',
     false
   ),
+  new ProjectInfo(
+    // TODO: Add token address here
+    '',
+    'CMARK-1',
+    '2022',
+    'LIMENET',
```

Comment:
This isn't accurate... the first project is going to be from OneEarth

WIP project page is here: https://app.carbonmark.com/projects/CMARK-1

Reply:
Done in ab38c93